src/Kernel32.Shared/Kernel32+CreateFileFlags.cs

```diff
@@ -146,13 +145,13 @@ public enum CreateFileFlags : uint
             SECURITY_SQOS_PRESENT = 0x00100000,
 
             /// <summary>Impersonates a client at the Anonymous impersonation level.</summary>
-            SECURITY_ANONYMOUS = SECURITY_IMPERSONATION_LEVEL.SecurityAnonymous << 16,
+            SECURITY_ANONYMOUS = 0 << 16,
```

Comment:
I do agree and think so also.

By optimized I meant disregarded by the compiler because it is the default value. It would only have readability meaning. I might have expressed myself on bad terms, but I don't know any better technical term to express this behavior other than "optimization" (the compiler disregarding the default values when they are explicit set).